public/API/API_GetAchievementOfTheWeek.php

```diff
@@ -47,66 +49,52 @@
  *       filters the Unlocks to just those entries after StartAt.
  */
 
-$achievementOfTheWeek = EventAchievement::active()
-    ->whereHas('achievement.game', function ($query) {
-        $query->where('Title', 'like', '%of the week%');
-    })
-    ->with('achievement.game')
-    ->first();
+$staticData = StaticData::first();
+$data = (new BuildAchievementOfTheWeekDataAction())->execute($staticData);
```

Comment:
nit: Recommend renaming to `$aotwData` or something else more descriptive. In my first read of the code, I had already forgotten what `$data` was when I landed on line 91 (obviously, all variables are data):
```php
'ID' => $data->forumTopicId->resolve() ?? null,
```